[2025-09-08T17:11:54.546+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-08T17:11:54.553+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskBranch backfill__2025-05-04T00:00:00+00:00 [queued]>
[2025-09-08T17:11:54.557+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskBranch backfill__2025-05-04T00:00:00+00:00 [queued]>
[2025-09-08T17:11:54.557+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-09-08T17:11:54.568+0000] {taskinstance.py:2888} INFO - Executing <Task(BranchPythonOperator): taskBranch> on 2025-05-04 00:00:00+00:00
[2025-09-08T17:11:54.572+0000] {standard_task_runner.py:72} INFO - Started process 22442 to run task
[2025-09-08T17:11:54.574+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'cron_catchup_backfill', 'taskBranch', 'backfill__2025-05-04T00:00:00+00:00', '--job-id', '220', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/executing_cron_catchup_backfill.py', '--cfg-path', '/tmp/tmp3zv32160']
[2025-09-08T17:11:54.575+0000] {standard_task_runner.py:105} INFO - Job 220: Subtask taskBranch
[2025-09-08T17:11:54.604+0000] {task_command.py:467} INFO - Running <TaskInstance: cron_catchup_backfill.taskBranch backfill__2025-05-04T00:00:00+00:00 [running]> on host codespaces-9bd7b3
[2025-09-08T17:11:54.655+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='loonycorn' AIRFLOW_CTX_DAG_ID='cron_catchup_backfill' AIRFLOW_CTX_TASK_ID='taskBranch' AIRFLOW_CTX_EXECUTION_DATE='2025-05-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2025-05-04T00:00:00+00:00'
[2025-09-08T17:11:54.655+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-08T17:11:54.667+0000] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-09-08T17:11:54.672+0000] {python.py:240} INFO - Done. Returned value was: taskD
[2025-09-08T17:11:54.672+0000] {branch.py:38} INFO - Branch into taskD
[2025-09-08T17:11:54.672+0000] {skipmixin.py:230} INFO - Following branch {'taskD'}
[2025-09-08T17:11:54.676+0000] {skipmixin.py:278} INFO - Skipping tasks [('taskC', -1), ('taskE', -1)]
[2025-09-08T17:11:54.695+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-08T17:11:54.695+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cron_catchup_backfill, task_id=taskBranch, run_id=backfill__2025-05-04T00:00:00+00:00, execution_date=20250504T000000, start_date=20250908T171154, end_date=20250908T171154
[2025-09-08T17:11:54.747+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-08T17:11:54.761+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-08T17:11:54.767+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
